README.md

```diff
@@ -430,6 +430,197 @@ QueryOptions options = QueryOptions.builder()
+        FaunaConfig config = new FaunaConfig.Builder()
+                .secret("FAUNA_SECRET")
+                .build();
+        FaunaClient client = Fauna.client(config);
+
+        // Use `feed()` to create an event feed with a blocking request.
+        FeedIterator<Product> syncIterator = client.feed(
+            fql("Product.all().eventsOn(.price, .stock)"),
+            Product.class // Class for the type returned in events
+        );
+
+        // Handle each page of events
+        syncIterator.forEachRemaining(page -> {
+            // Handle each event
+            for (StreamEvent<Product> event : page.getEvents()) {
+                System.out.println("Event: " + event);
+            }
+        });
+
+        // Get an event source.
+        Query query = fql("Product.all().eventSource() { name, stock }");
+        QuerySuccess<StreamTokenResponse> eventSourceResponse = client.query(query, StreamTokenResponse.class);
+        String eventSource = eventSourceResponse.getData().getToken();
+
+        // Create a FeedRequest.
+        FeedRequest feedRequest = FeedRequest.builder(eventSource)
+            .pageSize(10)
+            .build();
+
+        // Use `asyncFeed()` to get an event feed with a non-blocking request.
+        CompletableFuture<FeedIterator<Product>> futureFeed = client.asyncFeed(
+            feedRequest,
+            Product.class // Class for the type returned in events
+        );
+
+        futureFeed.thenAccept(iterator -> {
+            // Flatten pages to a single list of events.
+            List<StreamEvent<Product>> events = iterator.flatten().collect(Collectors.toList());
+            System.out.println("Received " + events.size() + " events from asyncFeed.");
+        }).exceptionally(ex -> {
+            System.err.println("Error initializing async feed: " + ex.getMessage());
+            return null;
+        });
+    }
+}
+```
+
+If changes occur between the creation of the event source and the request, the
+feed replays and emits any related events. In most cases, you’ll get events
+after a specific start time or event cursor.
+
+### Get events after a specific start time
+
+When you first poll an event source using an Event Feed, you usually pass
+a `startTs` argument to `feed()` or `asyncFeed()`.
+
+`startTs` is an integer representing a time in microseconds since the Unix
+epoch. The request returns events that occurred after the specified timestamp
+(exclusive).
+
+```java
+// Get an event source.
+Query query = fql("Product.all().eventSource() { name, stock }");
+QuerySuccess<StreamTokenResponse> eventSourceResponse = client.query(query, StreamTokenResponse.class);
+String eventSource = eventSourceResponse.getData().getToken();
+
+// Calculate the timestamp for 10 minutes ago in microseconds.
+long tenMinutesAgo = System.currentTimeMillis() * 1000 - (10 * 60 * 1000 * 1000);
+
+// Create a FeedRequest.
+FeedRequest feedRequest = FeedRequest.builder(eventSource)
+    .startTs(tenMinutesAgo)
```

Comment:
Q: could we have a more idiomatic overload here using `java.time` instead of a long?

Reply:
This got moved over to `FeedOptions` and morphed into a `Duration` in the process.